### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by rnugraha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,53 +84,37 @@ To ensure everything worked, run:
...
-    docker-compose -f src/main/docker/app.yml up -d
-
-For more information refer to [Using Docker and Docker-Compose][], this page also contains information on the docker-compose sub-generator (`yo podium:docker-compose`), which is able to generate docker configurations for one or several JHipster applications.
+To set up a CI environment, consult the [Setting up Continuous Integration][] page.
 
-### <a href="continuous-integration"></a>Continuous Integration (optional)
+## Wiki
 
-To set up a CI environment, consult the [Setting up Continuous Integration][] page.
+For more information please [see the wiki](https://github.com/thehyve/podium/wiki)
```
Comment (gijskant) [inline README.md]: Why wiki?

Comment (rnugraha) [reply README.md]: More extensive documentations

File: podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java [hunk 2]
```diff
@@ -577,4 +577,32 @@
...
 
+    /**
+     * Accept external request data and create a new request draft
+     *
+     * @return redirect to request form with filled in data
+     */
+    @PostMapping("/requests/external/new")
+    @SecuredByAuthority({AuthorityConstants.RESEARCHER})
+    @Timed
+    public ResponseEntity<Object> externalRequest(
```
Comment (gijskant) [inline podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java]: Why Object? What is the type?

File: podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java [hunk 3]
```diff
@@ -577,4 +577,32 @@
...
+        @RequestBody ExternalRequestRepresentation externalRequestRepresentation)
+        throws URISyntaxException, ActionNotAllowed {
+        AuthenticatedUser user = securityService.getCurrentUser();
+
+        log.debug("Create new external draft for user: {}\nWith data: {}", user, externalRequestRepresentation);
+
+        RequestRepresentation draft = draftService.createDraft(user);
+        Map<String, Object> result = requestService.createExternalRequest(draft, user, externalRequestRepresentation);
```
Comment (gijskant) [inline podium-gateway/src/main/java/nl/thehyve/podium/web/rest/RequestResource.java]: Okay... could you create a representation class for this?

File: podium-gateway/src/main/java/nl/thehyve/podium/service/RequestService.java [hunk 4]
```diff
@@ -576,4 +578,65 @@ public RequestRepresentation closeRequest(
...
 
+    /**
+     * Handle external request data and create a new request draft
+     *
+     * @param newRequest a newly initiated Request
+     * @param user Authenticated user making the request
+     * @param externalRequestRepresentation the data given to us from the external party
+     * @return Filled in RequestRepresentation
+     */
+     public Map<String, Object> createExternalRequest(RequestRepresentation newRequest, AuthenticatedUser user,
```
Comment (gijskant) [inline podium-gateway/src/main/java/nl/thehyve/podium/service/RequestService.java]: Please type the result.

File: podium-common/src/main/java/nl/thehyve/podium/common/service/dto/ExternalRequestRepresentation.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import java.util.Map;
+
+@Data
+public class ExternalRequestRepresentation implements Serializable{
+
+    @JsonProperty("URL")
+    String url;
+    String humanReadable;
+
+    ArrayList<Map<String, String>> collections;
```
Comment (gijskant) [inline podium-common/src/main/java/nl/thehyve/podium/common/service/dto/ExternalRequestRepresentation.java]: ArrayList? And I miss documentation here. What are these fields for, what do they mean?

**Pull request closed without merging**